test/lib/absinthe/relay/node/parse_ids_test.exs

```diff
@@ -662,4 +662,167 @@ defmodule Absinthe.Relay.Node.ParseIDsTest do
       assert {:ok, %{data: %{"updateParentLocalMiddleware" => expected_parent_data}}} == result
     end
   end
+
+  describe "find_schema_root!/2" do
```

Comment:
Hey @jacobparry thanks for taking a stab at testing this. Unfortunately this isn't the kind of test I had in mind. Absinthe.Relay could change its internal implementation of how it does the `{:fill` annotations within real schemas and these tests would continue to pass because the structures are hard coded.

I think the right way to test this is to create a real test schema with sub fields that use `parseIds`. Those should not explode when used. By testing the expected concrete behaviour it will prove that we've solved the bug, and be a proper safe guard against regressions.

Reply:
@benwilson512 Alright, check it out now. I believe that I have a test that fulfills the spirit and hopefully letter of what you are wanting accomplished.